libs/api/libbpf_link.cpp

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation
```

Comment:
are we not going to annotate these with SAL?

Reply:
We are including libbpf.h from libbpf repo. So we cannot modify the header file.

Reply:
Right